slasher/db/kv/highest_attestation.go

```diff
@@ -0,0 +1,81 @@
+package kv
+
+import (
+	"context"
+	"fmt"
+	json "github.com/json-iterator/go"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/slasher/detection/attestations/types"
+	bolt "go.etcd.io/bbolt"
+	"go.opencensus.io/trace"
+)
+
+func persistHighestAttestationCacheOnEviction(db *Store) func(key interface{}, value interface{}) {
+	// We use a closure here so we can access the database itself
+	// on the eviction of a span map from the cache. The function has the signature
+	// required by the ristretto cache OnEvict method.
+	// See https://godoc.org/github.com/dgraph-io/ristretto#Config.
+	return func(key interface{}, value interface{}) {
+
```

Comment:
if you don't write on eviction you never write cache into db

Reply:
yep I know, not implemented yet.